### ISSUE TITLE
Default Theme Color Problem

### ISSUE THREAD

**Issue by bajrangCoder** [TomSchimansky/CustomTkinter#741]
Is there any way to change the default color theme of app like the dark to light by option menu in any window

**Comment by Akascape** [comment]
Please refer this example: https://github.com/TomSchimansky/CustomTkinter/blob/master/examples/complex_example.py

**Comment by bajrangCoder** [comment]
> 

Yes, i have done same as dark, light theme switcher for color switcher but onchage default Theme Color ,after restarting it changes...

Any solution?
